bin/ridgepole

```diff
@@ -86,6 +86,19 @@ ARGV.options do |opt|
     opt.on('-f', '--file SCHEMAFILE') { |v| file = v }
     opt.on('',   '--dry-run') { options[:dry_run] = true }
     opt.on('',   '--table-options OPTIONS') { |v| options[:table_options] = v }
+    opt.on('',   '--table-hash-options OPTIONS') do |v|
+      # NOTE: Ruby2.4 doesn't support `symbolize_names: true`
+      hash = YAML.safe_load(v).deep_symbolize_keys
```

Comment:
I'm not planning to end support for Ruby 2.4 at this time, so could you use the code below?

```suggestion
      hash = YAML.safe_load(v, symbolize_names: true)
```